### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by H0MESlCK** [community/community#147952] [Other Feature Feedback, Questions, & Ideas]

**Comment by ShiiiivanshSingh** [comment]
same issue. any fix?

**Comment by TrueBIM** [comment]
Dear Community,

I am using my partner's account to post the following.

On 24/12/2024 I have created a new free account for working as part of a new role I recently started in a new company the organization name for my company and one new repo and one new project. After two days, I created the above, I tried to log in and I accidentally entered my password three times wrong. I immediately received the following screen. 
![image](https://github.com/user-attachments/assets/fb50bab7-f5dd-4b5e-8f9b-4d39d33a4555)
I then tried multiple times to recover my password using the "forgot password?" link, however I received no emails. I tried to log in with my correct password and I am navigated again at the above "Account suspended" screen.

On 27/12/2024, I opened a ticket using the https://support.github.com/contact/reinstatement page. I provided my personal mobile phone number which is not associated to any other Github accounts, and I have received the following email:
"
Thank you for contacting GitHub Support. We wanted to let you know that we've received your message.
Please note, we are receiving higher volumes of support requests and as a result you may experience longer wait times. In the meantime, you might find the answer to your question or a self-service solution to your problem in the [GitHub Docs](https://docs.github.com/). Feel free to check it out!
You can also post a question for other GitHub users in the [GitHub Community](https://github.community/). The Community can't help with account issues, like [reinstatement requests](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement), or with getting your ticket looked at faster.
Ticket ID: XXXXXXX
If you would like to add any additional information to your ticket, feel free to reply to this email.
Thank you!
The GitHub Support Team
 "  

Since that time, I haven't received any email back from Github. I have also replied back to them twice (replying to their email), sending kindly requests for looking into my case. However, I haven't heart from them yet.

I tried to submit a second ticket referencing my previous ticket and I the following message appeared. Weirdly, I haven't had any communication from them yet. I have also checked my junk email folder.
![image](https://github.com/user-attachments/assets/32afe01d-92b0-41be-8632-0e58b8bb66a9)

The problem I am having is that the email provided from my company (I recently started working) is now occupied and cannot be used at all even when trying to connect to other accounts. i.e. I tried to create a new Github account but also tried to add the email to my personal Github account - both tries were unsuccessful.

I am desperate and cannot have my company's email connected to a Github account at the moment. This makes me not able to be added to 
organization / repos / projects and I am not .

Is there anything else I could do to get my account recovered?

Any help would be appreciated.
Many thanks

**Comment by shinybrightstar** [comment]
Hi @NENE-01  

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
